src/lib/Filter/VersionFilter.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace EzSystems\HybridPlatformUi\Filter;
+
+use eZ\Publish\API\Repository\Values\Content\VersionInfo;
+
+class VersionFilter
+{
+    public function filterDrafts(array $versions)
```

Comment:
those public methods and the class deserve to have some documentation (also the file is missing the standard header)